tests/test_physics_benchmarking.py

```diff
@@ -1,8 +1,10 @@
 import pytest
 
 import utils
+from habitat_sim.bindings import vhacd_enabled
```

Comment:
Calling bindings from the test is not recommended. I am surprised the import linter didn't catch it.

Reply:
Why is it not a good idea? And what would the alternative be?

Reply:
Nothing should be imported from bindings, it should have a dedicated namespace somewhere. 
```import habitat_sim
habitat_sim.vhacd_enabled
```
or
```
 from habitat_sim.vhacd import vhacd_enabled 
 ```
 
 We don't want the users to use bindings, it's mostly there for backwards compat.